### DISCUSSION TITLE
Should there be a standard in libre localization?

### DISCUSSION THREAD

**Discussion by orangesunny** [WeblateOrg/weblate#8743] [Ideas]
This is a place to discuss your ideas and efforts relating to a possible Translation.md or a similar standard to guide translators of libre software to the correct place where they can localize their favorite tools and bring it theirs :)

It starts with a lightning talk on FOSDEM 2023:
https://fosdem.org/2023/schedule/event/standards_in_libre_localization/

**Comment by Mark-Giesen** [comment]
There might! We have a TRANSLATING.md file in our repository. It's purpose is mainly to be a quick guide to using Weblate to help the teachers that use our software to quickly find there way around Weblate to fix translation errors and add new translations as they need them. 
I guess you and Dwayne were aiming at a different audience: translators finding a project? You talked about a 9 page draft of the idea for a standard, is this shared somewhere? Our idea and intention, might be one of the items on those 9 pages, or could be added.

**Comment by comradekingu** [comment]
While not libre-specific, `TRANSLATING.md` has been a/the way to do it since whenever I recommended it for F-Droid.
Probably still undocumented, but it gets picked up and added as a link for the respective software entry.
It is also possible to do it in the F-Droid-specific metadata.

**Comment by nijel** [comment]
Just for the reference, here is what kind of standard files are supported at GitHub: https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file#supported-file-types
